packages/host/app/components/operator-mode/stack-item.gts

```diff
@@ -427,6 +440,14 @@ export default class OperatorModeStackItem extends Component<Signature> {
     this.containerEl.scrollTop = 0;
   });
 
+  private doCloseAnimation = task(async () => {
+    this.isClosing = true;
+    if (!isTesting()) {
+      // wait for the animation to complete
+      await timeout(100);
```

Comment:
I tried the approach but it can't achieve the anitmation to dismiss the last card, here is my example from another branch: https://github.com/cardstack/boxel/commit/b090abe58dc4d989d3e48af7666b5bcbd5284531

Reply:
so i think you need to use Promise.all like this:
```
await Promise.all(
  elem.getAnimations({ subtree: true }).map((animation) => animation.finished),
);
```

(from https://developer.mozilla.org/en-US/docs/Web/API/Element/getAnimations)